src/Plugin.InAppBilling/InAppBilling.android.cs

```diff
@@ -9,6 +9,11 @@
 using System.Text;
 using Android.BillingClient.Api;
 using Android.Content;
+#if NET6ANDROID
```

Comment:
Why not just `#if NET` (it's defined only for net5 and net6)

Reply:
There're a lot of different symbols now. Even `#if ANDROID` would work here

https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/preprocessor-directives#conditional-compilation